scripts/add-crowdin-contributors.ts

```diff
@@ -11,7 +11,7 @@ import allContributors from 'all-contributors-cli';
  * and paste its output into this script.
  *
  * Usage:
- * 1. Open https://crowdin.com/project/getferdi/settings#members
+ * 1. Open https://crowdin.com/project/ferdium/settings#members
```

Comment:
This link is also dead, after the fix for the recipes, we may want to revive the Crowdin project (translation server).